data/assets/scene/solarsystem/heliosphere/bastille_day/fluxnodeslegend.asset

```diff
@@ -0,0 +1,35 @@
+local TexturesPath = asset.syncedResource({
+    Type = "HttpSynchronization",
+    Name = "Streamnodes textures",
+    Identifier = "streamnodes_legend",
+    Version = 1
+})
+
+asset.require("spice/base")
+local assetHelper = asset.require("util/asset_helper")
+--local textures = asset.require('./streamnodes_textures').TexturesPath
+
+local legend = {
+    Identifier = "Legendfluxnodes",	
+    --Type = "ScreenSpaceRenderable",
+    Type = "ScreenSpaceImageLocal",
+    TexturePath = TexturesPath .. "/CMR_transparent_vittext.png",
+};
+
+openspace.addScreenSpaceRenderable(legend)
+openspace.setPropertyValueSingle('ScreenSpace.Legendfluxnodes.CartesianPosition', {0.677970, 0.0,-1.049180});
+-- openspace.setPropertyValueSingle('ScreenSpace.Legendfluxnodes.CartesianPosition', {0.915250,-0.033900,-1.508200});
```

Comment:
why is this here? remove? 

Also, could maybe be good with a comment explaining where the values in all these `setPropertyvalueSinge`s come from